patches/additionalPythonPackages/default.nix

```diff
@@ -1121,4 +1121,80 @@ EOF
     };
   };
 
+  rpy2 = pythonPackages.buildPythonPackage rec {
```

Comment:
Ah ! So, patching rpy2 solved yesterday's problem ? Did you copy pasted the one from NixOS ?

Reply:
Kind of copy/paste with some fixes to resolve some names.